docs/content/page/index.en-us.md

```diff
@@ -11,22 +11,21 @@ draft: false
 # What is Gitea?
 
 Gitea is a painless self-hosted Git service. It is similar to GitHub, Bitbucket, and GitLab.
-Gitea is a fork of [Gogs](http://gogs.io). See the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
+Gitea is a fork of repository named [Gogs](http://gogs.io). Kindly check out the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
```

Comment:
While it's also true that the Gitea repo is a fork of the Gogs repo, the previous statement makes more sense imo, that the Gitea project is a fork of the Gogs project, or "Gitea is a fork of Gogs".

I do like the change to the next sentence, though. 🙂

Reply:
ohkk....thank you for considering 😀